lib/loader.js

```diff
@@ -2,18 +2,39 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
+var path = require("path");
 var loaderUtils = require("loader-utils");
 var processCss = require("./processCss");
 var getImportPrefix = require("./getImportPrefix");
 var compileExports = require("./compile-exports");
 
+/**
+ * If the file was "renamed" (for the purposes of source maps), then honor it,
+ * otherwise just use the resourcePath.
+ * @param {String} resourcePath - The absolute file system path for the sass file.
+ * @param {Object|null} map - An existing source map, if any.
+ * @return {String} - The effective path to use for the `from` argument.
+ */
+function processFrom(resourcePath, map) {
+	var effectiveResourcePath;
+	if (map && map.file && typeof map.file === 'string' && path.dirname(map.file) === '.') {
```

Comment:
`path.dirname(map.file) === '.'`  checks for a relative path here ?  I'm not sure I'm following.. 😛 . In case my understanding is correct here, there is also `path.isAbsolute(filepath)` available

Reply:
I've done some thinking on it and here I may be trying to be too clever. What I originally was thinking of shouldn't be something that css-loader is responsible for. I can provide more details on what I was thinking if you'd like.

I'll try cleaning this up in my next commit.

```javascript
		from: this.resourcePath,
		to: this.resourcePath,
```

will be enough.